client/src/app/index.tsx

```diff
@@ -20,9 +34,17 @@ const App = () => {
   }, [segments, navigationState?.key, initialized]);
 
   return (
+    <SettingsContext.Provider value={{
+      theme,
+      isSendButtonDisabled,
+      autoCorrect,
+      autoCapitalize,
+      headerBackgroundColor
+    }}> 
```

Comment:
:star: Tip! index.tsx doesn't actually render anything! or wrap anything as its children!

Wrap the _layout.tsx responsible for the chat screen within `(home)/_layout.ts`!